### DISCUSSION TITLE
extended_auditing_policy remediation NOT working

### DISCUSSION THREAD

**Discussion by wednesday09** [aquasecurity/trivy#5738] [Bugs]
### Description

This is the remediation action for enabling audit on Azure SQL Databases and it's not working

LINK: https://avd.aquasec.com/misconfig/azure/database/avd-azu-0027
------------------------------------
I saw these 2 links for your reference:
https://github.com/hashicorp/terraform-provider-azurerm/issues/19376
https://github.com/hashicorp/terraform-provider-azurerm/pull/19390

### Desired Behavior

extended_auditing_policy block should work

### Actual Behavior

I got an error saying: Unexpected block: Blocks of type "extended_auditing_policy" are not expected here

### Reproduction Steps

```bash
resource "azurerm_sql_server" "good_example" {
   name                         = "mssqlserver"
   resource_group_name          = azurerm_resource_group.example.name
   location                     = azurerm_resource_group.example.location
   version                      = "12.0"
   administrator_login          = "mradministrator"
   administrator_login_password = "tfsecRocks"
 
   extended_auditing_policy {
     storage_endpoint                        = azurerm_storage_account.example.primary_blob_endpoint
     storage_account_access_key              = azurerm_storage_account.example.primary_access_key
     storage_account_access_key_is_secondary = true
     retention_in_days                       = 6
   }
 }

This is the sample on your recommended action, but the extended_auditing_policy don't work.
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
No debug output
```

### Operating System

Windows 10

### Version

```bash
1.5.4
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Hi @wednesday09 !

Thanks for the report. I created issue https://github.com/aquasecurity/trivy/issues/5739
